web/src/test/java/org/cbioportal/genome_nexus/web/AnnotationIntegrationTest.java

```diff
@@ -67,12 +67,17 @@ public void testAnnotation()
         //////////////////
 
         // for each variant we should have one matching instance, except the invalid one 
```

Comment:
update this comment to reflect the changes since we no longer remove the invalid variant from the list of responses here